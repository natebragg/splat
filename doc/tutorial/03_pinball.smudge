// 03_pinball.smudge
pinball
{
   * idle
     [
          coin --> run
     ],
     run
     [
          target -(@incScore)-,
          drain --> idle
     ],
}
