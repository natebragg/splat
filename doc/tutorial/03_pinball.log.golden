ding ding, new score: 1
ding ding, new score: 2
